client/lib/api/user_preferences.dart

```diff
@@ -92,8 +95,26 @@ class UserPreferences {
   }
 
   Future<bool> setNotificationsEnabled(bool value) async {
```

Comment:
@epicfaace - do you know if this is called when someone updates the system settings? For example if they disable notifications in the iOS system settings... but leave it enabled within the app, it would be confusing. If that's the case, please file a bug on it. Not a priority to fix for v1 but I think something we should track for the future.

Reply:
I don't think so -- will file a bug

Reply:
See https://github.com/WorldHealthOrganization/app/issues/1741